blockchain/blockchain-wallet-v4-frontend
# fix(lang): remove italian and german translations

## Description (optional)
Remove Italian and German translations as they are no longer supported.

## Testing Steps (optional)
Login -> Preferences -> confirm that Italian and German are not available options in the Wallet language dropdown
